docs/user_guide/data_types_and_io/dataclass.md

```diff
@@ -11,8 +11,31 @@ When you've multiple values that you want to send across Flyte entities, you can
 Flytekit uses the [Mashumaro library](https://github.com/Fatal1ty/mashumaro)
 to serialize and deserialize dataclasses.
 
+With the 1.14 release, `flytekit` adopted `MessagePack` as the 
+serialization format for dataclasses, overcoming  a major limitation of serialization into a JSON string within a Protobuf `struct` datatype, like the previous versions do:
+
```

Comment:
```suggestion
```

Reply:
I think new line is more readable
<img width="675" alt="image" src="https://github.com/user-attachments/assets/b6785572-35ec-4936-b901-96984ef1f587">